src/HoneybadgerServiceProvider.php

```diff
@@ -92,6 +98,11 @@ private function bindCommands()
             HoneybadgerCheckinCommand::class
         );
 
+        $this->app->bind(
```

Comment:
I'm not 100% certain, but I don't think this is needed. I think you can just pass the list of command classes to `$this->commands`.

Reply:
I just followed the way other commands are defined here 🤷‍♂️